src/pyflexplot/config/settings.yaml

```diff
@@ -5,10 +5,16 @@ main:
       input:
         region: eu-central-2
         name: meteoswiss-flexpart-flexpart-output-dev
+        endpoint_url: ""
+        s3_access_key: ""
+        s3_secret_key: ""
```

Comment:
I like the endpoint_url added to the config but not so much the s3_access_key/secret_key. These can be provided as environment variables or mounted as a file to ~/.aws/credentials

Reply:
I removed them from the config and provided them as environment variables in my .env in my flexpart orchestrator repo.